### PR TITLE
xuperchain upgrade to new framework

### PR DESCRIPTION
**Pull request by wellzm**
## Description

What is the purpose of the change?
xuperchain upgrade to new framwork based on xupercore

Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Brief of your solution

xuperchain upgrade to new framwork based on xupercore

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### CODE REVIEW

File: conf/server.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# rpcPort service listen port for xuperos
+rpcPort: 36201
```
Comment (qizheng09) [inline conf/server.yaml:2]: 这些默认参数建议与xuperchain之前一致呢。

File: cmd/client/cmd/account_new.go [hunk 2]
```diff
@@ -12,10 +12,9 @@ import (
 	"io/ioutil"
 
 	"github.com/spf13/cobra"
-
-	"github.com/xuperchain/xuperchain/core/pb"
-	"github.com/xuperchain/xuperchain/core/permission/acl/utils"
-	"github.com/xuperchain/xuperchain/core/utxo"
+	"github.com/xuperchain/xuperchain/common/xupospb/pb"
```
[the remark this answers is not in the document]
Comment (wellzm) [reply cmd/client/cmd/account_new.go:15]: ok

File: common/def/def.go [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package def
+
+const (
+	SubModName = "xuperos"
```
Comment (chunhui01) [inline common/def/def.go:4]: 可以使用xuperchain或者xchain。

File: common/utils/utils.go [hunk 4]
```diff
@@ -0,0 +1 @@
+package utils
```
Comment (chunhui01) [inline common/utils/utils.go:1]: 空包建议删除~

File: common/xupospb/xuperos.proto [hunk 5]
```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
```
Comment (chunhui01) [inline common/xupospb/xuperos.proto:1]: 这个proto还需要吗？不需要了可以删除哈

**Review by chunhui01 — APPROVED** [review]
修改较多，但大部分都是删除，关键代码在xuperos中得到过验证了，可以合入。后续需要整体回归测试。

**Review by icexin — APPROVED** [review]

**Pull request merged**